thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-details/template.hbs

```diff
@@ -95,8 +95,8 @@
         </div>
       {{/if}}
       {{#if (eq displayMode "list")}}
-        {{#link-to "manage.explore" alertData.id}}
-          <div class="te-search-results__title-name" title={{alertData.functionName}}>{{alertData.functionName}}</div>
+        {{#link-to "manage.explore" alertData.id class="te-search-results__title-link"}}
+          <p class="te-search-results__title-name" title={{alertData.functionName}}>{{alertData.functionName}}</p>
```

Comment:
This code is also in `.../self-serve-alert-yaml-details/template.hbs` 
should it also be changed so that they are consistent?

Reply:
Thanks for notifying me, could you guide me on which page uses `.../self-serve-alert-yaml-details/template.hbs`? Let me double-check it.

Reply:
`...pods/manage/explore/template.hbs` and
 `.../pods/manage/yaml/template.hbs` reference

 `{{#self-serve-alert-yaml-details}}`